calico_kubernetes/calico_kubernetes_cni.py

```diff
@@ -333,6 +333,10 @@ def _call_ipam_plugin():
     plugin_path = os.path.abspath(os.path.join(plugin_dir, plugin_type))
     _log.info("Using IPAM plugin at: %s", plugin_path)
 
+    if not os.path.isfile(plugin_path):
+        _log.error("File at %s does not exist. Exiting.", plugin_path)
```

Comment:
You can make this more specific (we know what we're looking for, it's not just any old file).  That will make it a better log message.

Instead of "File at...", you can say something like "Could not find IPAM plugin X at location Y".